fgpyo/fasta/sequence_dictionary.py

```diff
@@ -9,10 +9,10 @@
 >>> import pysam
 >>> from fgpyo.fasta.sequence_dictionary import SequenceDictionary
 >>> sd: SequenceDictionary
->>> with pysam.AlignmentFile("./fgpyo/sam/tests/data/valid.sam") as fh:
-    ...    sd = SequenceDictionary.from_sam(header=fh.header)
+>>> with pysam.AlignmentFile("./tests/fgpyo/sam/data/valid.sam") as fh:
```

Comment:
For clarity I might `SKIP` this and replace with a dummy path, as in the test for `FastxZipped` below

Reply:
I like this since it _actually_ runs!